doc/radosgw/account.rst

```diff
@@ -0,0 +1,129 @@
+===============
+ User Accounts
+===============
+
+.. versionadded:: Squid
+
+Ceph Object Gateway supports user accounts as an optional feature to enable
+the self-service management of `Users`_ and `Roles`_ similar to those in
+`AWS Identity and Access Management`_ (IAM).
+
+.. _radosgw-account-root-user:
+
+Account Root User
+=================
+
+Like normal users and roles, accounts and account root users must be
```

Comment:
Here you have "account root" but above "user accounts", should above be "root users" too?  "account" below as well, we should be consistent throughout the document.

Reply:
thanks @anthonyeleven, i tried to clarify the language here. would you mind giving the docs another review?